### DISCUSSION TITLE
Remember to make the exit button work.

### DISCUSSION THREAD

**Discussion by MoonlitBeans** [jgoll7/Tafe22S211#5] [General]
I completely forgot the exit button needs to work! I pushed and merged without a working exit button. :(

I'm going to fix that and update the branch. I'll merge after you this time. :)

**Comment by jgoll7** [comment]
Forgot about this too, thanks for the reminder! Did you also make the button on the MainMenu re-direct to the currency page?

**Reply by MoonlitBeans** [reply]
No, but I probably should do that. I completly forgot about that one too tbh
